File: UsaloYa.Dto/RegisterUserAndCompanyDto.cs
using System.ComponentModel.DataAnnotations;

namespace UsaloYa.Dto
{
    public class RegisterUserAndCompanyDto
    {
        [Required]
        public RequestRegisterNewUserDto RequestRegisterNewUserDto { get; set; }

        [Required]
        public CompanyDto CompanyDto { get; set; }

        [Required]
        public GroupDto GroupDto { get; set; }
    }
}

File: Guardian/src/app/dto/RegisterUserAndCompanyDto .ts
import { RequestRegisterNewUserDto } from "./RequestRegisterNewUserDto ";
import { companyDto } from "./companyDto";
import { GroupDto } from "./groupDto";

export interface RegisterUserAndCompanyDto {
  requestRegisterNewUserDto: RequestRegisterNewUserDto;
  companyDto: companyDto;
  groupDto: GroupDto;
}
